loadtest/control/simulcontroller/actions.go

```diff
@@ -2062,3 +2062,37 @@ func (c *SimulController) openPermalink(u user.User) control.UserActionResponse
 
 	return control.UserActionResponse{Info: fmt.Sprintf("clicked permalink on post %s", postID)}
 }
+
+func (c *SimulController) generateUserReport(u user.User) control.UserActionResponse {
```

Comment:
Would we want to test a bunch of scenarios here? To me likely the biggest load on the system would be to return as many users as possible, but are we wanting to test different sorting/filtering here too?

Reply:
> Would we want to test a bunch of scenarios here? To me likely the biggest load on the system would be to return as many users as possible, but are we wanting to test different sorting/filtering here too?


Ideally, we'd have a behaviour as similar as possible as that of a real server (on average during a high traffic time window). That's not possible yet, since the feature is not in production, so I'm happy covering the use case with the biggest load. Once the webapp side of the feature is done and we know how the interaction will look like, we can always iterate here.